docs/modules/ROOT/pages/cops_metrics.adoc

```diff
@@ -57,6 +57,8 @@ You can set literals you want to fold with `CountAsOne`.
 Available are: 'array', 'hash', and 'heredoc'. Each literal
 will be counted as one line regardless of its actual size.
 
+Note: This cop does not apply for struct definitions
```

Comment:
Note -> NOTE (that's the proper AsciiDoc syntax)

Don't forget to terminate the sentence with a `.`.